tasks/request_status_for_job/test/unit_tests/test_request_status_for_job_unit.py

```diff
@@ -17,30 +17,32 @@ class TestRequestStatusForJobUnit(unittest.TestCase):  # pylint: disable-msg=too
 
     # noinspection PyPep8Naming
     @patch('request_status_for_job.task')
+    @patch('requests_db.get_dbconnect_info')
     @patch('cumulus_logger.CumulusLogger.setMetadata')
     def test_handler_happy_path(
             self,
             mock_setMetadata: MagicMock,
+            mock_get_dbconnect_info: MagicMock,
             mock_task: MagicMock
     ):
         async_operation_id = uuid.uuid4().__str__()
 
         event = {
-            request_status_for_job.ASYNC_OPERATION_ID_KEY: async_operation_id
+            request_status_for_job.INPUT_JOB_ID_KEY: async_operation_id
         }
         context = Mock()
         result = request_status_for_job.handler(event, context)
 
         mock_setMetadata.assert_called_once_with(event, context)
-        mock_task.assert_called_once_with(async_operation_id)
+        mock_task.assert_called_once_with(async_operation_id, mock_get_dbconnect_info.return_value)
         self.assertEqual(mock_task.return_value, result)
 
```

Comment:
Possibly add test to validate schema output for a valid job_id.

Notice we test input ok but not necessarily output unless I am missing something.

Reply:
The handler does little more than pass properties through to the 'task'. It just returns whatever 'task' returns, so there isn't really anything to check in the output.

Reply:
I was thinking a separate test def test_valid_scema_output_scenarions....
I would probably employ something like https://github.com/Julian/jsonschema or https://horejsek.github.io/python-fastjsonschema/ to validate the output against our defined schema. That way you could always test/validate outputs. You could use it to validate inputs to as well. Just thinking there are no tests that actually validate the output from the lambda is validated against our defined schema.